.github/workflows/ci.yml

```
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11-dev"]
+        python-version: ["3.6", "3.7.0", "3.7", "3.8", "3.9", "3.10", "3.11-dev"]
```

```suggestion
        python-version: ["3.6", "3.7.1", "3.7", "3.8", "3.9", "3.10", "3.11-dev"]
```
3.7.0 apparently doesn't exist on GH Actions